### ISSUE TITLE
Skill Boost & PERFECT SUPPORT 

### ISSUE THREAD

**Issue by Y-li27** [superk589/DereGuide#23]
![f1f1bf76-8f2e-4f38-a3df-463c7e0c9f28](https://user-images.githubusercontent.com/32714391/36466414-749b91fe-171d-11e8-9974-ecbe67d3208f.png)
![1e613fdb-47d9-47c5-a887-c53d8bd192f0](https://user-images.githubusercontent.com/32714391/36466425-836be288-171d-11e8-87d7-384a7960d7f9.png)

I'd like you to add an option to skill boost and perfect support to let you calculate the organization you want to leave...
In short, I would like you to attach an option that supposes to assume all mistakes.

**Comment by superk589** [comment]
I think I've known what you'd like to do.
About developing, here I list all TODOs:
* Add support for note evaluation(perfect, great, good etc.), so that we can calculate score and life healed with notes other than only perfect notes.
* Calculate life loss when note is missed. The value is vary with difficulties and note types.
* Add "afk" mode, in this mode all notes will be missed unless skill boost and ssr's perfect support are on.
  * Maybe here we need two modes: afk mode and afk optimistic mode. In afk optimistic mode, all skills have 100% proc rate.
  * Maybe we also need a new calculate button, to calculate whether we can get S score rank in afk mode and how much the rate of surviving the live in afk mode is.

**Comment by superk589** [comment]
As v1.8 is released, close this issue.
